### PR TITLE
DPanel to DFrame

### PR DESCRIPTION
**Pull request by MrUnknownGamer**
Very tiny fix,  but it may help in the future when someone is trying to make something for the scoreboard or modify it.

### CODE REVIEW

**Comment by bmwalters** [comment]
But... the passed parameter in a lot of those situations IS a DPanel...

**Comment by Kefta** [comment]
It's just a variable name

File: garrysmod/gamemodes/terrortown/gamemode/cl_scoring.lua [hunk 1]
```diff
@@ -426,7 +426,7 @@ function CLSCORE:ClearPanel()
       -- we need this hack as opposed to just calling Remove because gmod does
       -- not offer a means of killing the tooltip, and doesn't clean it up
       -- properly on Remove
-      input.SetCursorPos( ScrW()/2, ScrH()/2 )
+      gui.SetMousePos( ScrW()/2, ScrH()/2 )
```
Comment (Bo98) [inline garrysmod/gamemodes/terrortown/gamemode/cl_scoring.lua]: Why are you reverting other people's changes?

Comment (markusmarkusz) [inline garrysmod/gamemodes/terrortown/gamemode/cl_scoring.lua]:
```LUA
gui.SetMousePos
´´´
is deprecated.
See #1205

**Comment by SpencerSharkey** [comment]
Wouldn't call this a "fix" 
DFrame = DPanel

**Comment by bmwalters** [comment]
@SpencerSharkey No, there is a huge difference between the two panel types. The problem with this PR is that A) it might be unnecessary, B) it changes some places where they were ACTUALLY DPanel to "dframe" variable name.

**Pull request closed without merging**